### ISSUE TITLE
Yahoo Mail Log-in

### ISSUE THREAD

**Issue by MarkSpizu** [sarahavilov/secure-login#35]
Secure Login (Embedded WebExtension) doesn't work well with Yahoo Mail Log-in. I always get this message 

Do you have access to this email?

m****nf@yahoo.com

We will send you an Account Key to verify you're the owner of this email. 

If I disable your addon this nonsense with Account Key disappears and I can log-in normally.

By the way, with the old Secure Login no problems.

Well, can you fix it?

**Comment by MarkSpizu** [comment]
Nobody answer to issues... it looks like a dead place...

**Comment by sarahavilov** [comment]
Seems like Yahoo checks to see if the form submission is automatic or with user interaction. Basically browsers have a flag when an even is dispatched that displays whether the event is initiated by user or not. The only workaround is to ask the extension to fill the form, but not to submit it.

**Comment by MarkSpizu** [comment]
Yeah thanks. The downside is that Submit Form will be disabled for all websites. I would have preferred to exclude Yahoo only from "Submit login form automatically".

**Comment by sarahavilov** [comment]
Okay then, we will have a new right-click context menu to fill forms without submission. As of the next release, you can reset the options page to submit forms and use the new right-click item only on Yahoo.

**Comment by MarkSpizu** [comment]
That's good to hear. I'm waiting for the next release. Thanks a lot!
